src/main/java/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -1135,6 +1135,20 @@ void fightAirRaidsAndStrategicBombing(final IDelegateBridge delegateBridge, Supp
     }
   }
 
+  public void fightAutoKills(final IDelegateBridge aBridge) {
+    // Kill undefended transports. Done first to remove potentially dependent sea battles
+    // Which could block amphibious assaults below
+    getPendingBattleSites(false).stream().map(territory -> getPendingBattle(territory, false, BattleType.NORMAL))
```

Comment:
styling nit. Please prefer to put one logical operation per line when using stream APIs, it helps readability. In this case for example  `.map(territory..` would be moved to the next line.